### DISCUSSION TITLE
How can I play sound effects?

### DISCUSSION THREAD

**Discussion by UndeRus** [mum-rs/mum#132] [Q&A]

**Comment by sornas** [comment]
They should already play automatically when e.g. connecting to a server. If you try to connect to a server, can you hear any sound at all? Or can you hear other clients but specifically not sound effects?

**Comment by UndeRus** [comment]
I want to:
1. Disable or config sound effects, which played on events 
2. Play sound effects to mumble by external call(like soundboard)

**Comment by sornas** [comment]
1 we can do. ill create a separate issue. a soundboard is out of scope for this project i think. you could look for some kind of virtual audio device program (i think i myself have used plain old pulseaudio for this) and a separate soundboard. i cant really help you with that though, sorry.
